### PR TITLE
APM-341627

### PR DESCRIPTION
**Pull request by kamiljsokolowski**
This change will:

1. Disable executing the pipeline on PR creation and updates.
2. Disable executing the pipeline on PRs coming from forks.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,7 +38,7 @@ jobs:
     - stage: build-package
       name: Building package & Deploying to Storage Account
       sudo: required
-      if: ((branch = master OR type = pull_request) AND fork = false) OR tag =~ /^release.*$/
+      if: branch = master OR tag =~ /^release.*$/
```
Comment (karolkieglerski) [inline .travis.yml:41]: it should run on all branches not only on master if you remove PR

File: .travis.yml [hunk 2]
```diff
@@ -52,7 +53,7 @@ jobs:
       language: python
       python:
         - "3.8"
-      if: (branch = master OR type = pull_request) AND fork = false
+      if: branch = master OR tag =~ /^release.*$/
```
Comment (karolkieglerski) [inline .travis.yml:56]: it should run on all branches not only on master if you remove PR

**Review by karolkieglerski — CHANGES_REQUESTED** [review]
remove branch = master

File: .travis.yml (hunk 1 above)
Comment (kamiljsokolowski) [reply .travis.yml]: Done.

File: .travis.yml (hunk 2 above)
Comment (kamiljsokolowski) [reply .travis.yml]: Done.

**Review by karolkieglerski — APPROVED** [review]

**Pull request merged**